phpstan.neon

```diff
@@ -25,3 +25,4 @@ parameters:
     - Configuration/Backend/AjaxRoutes.php
     - Classes/Updates/Typo3Lts8Update.php
     - Classes/Controller/Backend/Update/*
+    - Classes/Controller/Backend/ControlCenter/Update/*
```

Comment:
This controller will leave us, after solving https://github.com/T3Voila/templavoilaplus/issues/372

Reply:
I know, but as long as it exists, it needs to be resolvable.

Reply:
Ok, cleaned that part up now, the line "Classes/Controller/Backend/Update/*" could IMHO also removed

Reply:
Thus reduced everything to allow the command to update the stan baseline to remove warnings because errors are fixed